src/java.base/share/classes/java/lang/invoke/InvokerBytecodeGenerator.java

```diff
@@ -182,8 +183,8 @@ private static String makeDumpableClassName(String className) {
         }
         String sfx = ctr.toString();
         while (sfx.length() < 3)
-            sfx = "0" + sfx;
-        className += sfx;
+            sfx = "0".concat(sfx);
```

Comment:
This can probably be
```java
if (sfx.length() < 3)
    sfx = "0".repeat(3 - sfx.length()).concat(sfx);
```

Reply:
The repeat method will throw an error when count is negative. I refactored the entire method to repeat('0', 2) when count is < 10 and append('0') when count is < 100, which is clearer.